configs/3playground.json

```
@@ -8,18 +8,17 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "section.section h2",
+      "selector": "div.section h1",
       "global": true,
       "default_value": "Documentation"
     },
-    "lvl1": "section.section h3",
-    "lvl2": "section.section h4",
-    "lvl3": "section.section h5",
-    "lvl4": "section.section h6",
-    "text": "section.section p, section.section li"
+    "lvl1": "div.section h2",
+    "lvl2": "div.section h3",
+    "lvl3": "div.content h3, div.content h4, div.content h5, div.section h3, div.section h4, div.section h5", 
+    "text": "section.section p, section.section li, div.content p, div.section p, div.content li, div.section li"
```

```suggestion
    "text": ".section p, .section li, .content p"
```
Is this suggested selector matching the same elements? It will help to avoid redundancy within the selector